### PR TITLE
Update 4DM2 for Semifinals

### PR DESCRIPTION
**Pull request by fajar13k**
- Add Semifinals mappool
- Add Semifinals match schedule
- Add Quarterfinals match result
- Organisation update

### CODE REVIEW

File: wiki/Tournaments/4DM/2/en.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ The **4 Digit osu!mania World Cup 2** (***4DM2***) is a worldwide country-based
 
 ![](img/badge.png "4DM2 winner badge")
 
+Generous donators coming from ![][flag_DO] [Antalf](https://osu.ppy.sh/users/8793773) and ![][flag_NL] [NightNarumi](https://osu.ppy.sh/users/4381142) whom which made up some of the prizes.
```
Comment (cl8n) [inline wiki/Tournaments/4DM/2/en.md]:
```suggestion
Generous donations from ![][flag_DO] [Antalf](https://osu.ppy.sh/users/8793773) and ![][flag_NL] [NightNarumi](https://osu.ppy.sh/users/4381142) helped fund some of the prizes.
```

File: wiki/Tournaments/4DM/2/en.md [hunk 2]
```diff
@@ -103,8 +106,58 @@ The 4 Digit osu!mania World Cup 2 is run by various community members.
...
+| :-: | --: | :-: | :-: | :-- | :-: |
+| #21a¹ | France | ![][flag_FR] | ![][flag_TH] | Thailand | **12:00** |
+| #21b¹ | France | ![][flag_FR] | ![][flag_SG] | Singapore | **12:00** |
+| #21c¹ | Brazil | ![][flag_BR] | ![][flag_TH] | Thailand | **12:00** |
+| #21d¹ | Brazil | ![][flag_BR] | ![][flag_SG] | Singapore | **12:00** |
+| #22a² | Russian Federation | ![][flag_RU] | ![][flag_AR] | Argentina | **13:00** |
+| #22b² | Russian Federation | ![][flag_RU] | ![][flag_CA] | Canada | **13:00** |
+| #22c² | South Korea | ![][flag_KR] | ![][flag_AR] | Argentina | **13:00** |
+| #22d² | South Korea | ![][flag_KR] | ![][flag_CA] | Canada | **13:00** |
+| #23 | United Kingdom | ![][flag_GB] | ![][flag_US] United States | **18:00** |
```
Comment (cl8n) [inline wiki/Tournaments/4DM/2/en.md]:
```suggestion
| #23 | United Kingdom | ![][flag_GB] | ![][flag_US] | United States | **18:00** |
```

File: wiki/Tournaments/4DM/2/en.md [hunk 3]
```diff
@@ -103,8 +106,58 @@ The 4 Digit osu!mania World Cup 2 is run by various community members.
...
+
+¹ Encounter dependent on winner of Match #17 and Match #18
```
Comment (cl8n) [inline wiki/Tournaments/4DM/2/en.md]:
```suggestion
¹ Encounter dependent on winner of Match #17 and Match #18  
```
the two spaces at the end tell markdown to add a newline

Comment (fajar13k) [reply wiki/Tournaments/4DM/2/en.md]: That's new to me

File: wiki/Tournaments/4DM/2/en.md [hunk 4]
```diff
@@ -168,6 +221,26 @@ The 4 Digit osu!mania World Cup 2 is run by various community members.
...
+| **Philippines** ![][flag_PH] | **5** | 2 | ![][flag_AR] Argentina | [#1](https://osu.ppy.sh/community/matches/58993495) |
+
+Sunday, 8 March 2020:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Brazil** ![][flag_BR] | **5** | 1 | ![][flag_AU] Australia | [#1](https://osu.ppy.sh/community/matches/59012301) |
+| **Canada** ![][flag_CA] | **5** | 0 | ![][flag_NZ] New Zealand | [#1](https://osu.ppy.sh/community/matches/59012904) |
+| **South Korea** ![][flag_KR] | **5** | 0 | ![][flag_NL] Netherlands | [#1](https://osu.ppy.sh/community/matches/59027342) |
+| Russian Federation ![][flag_CN] | 1 | **5** | ![][flag_CN] **China** | [#1](https://osu.ppy.sh/community/matches/59027346) |
```
Comment (cl8n) [inline wiki/Tournaments/4DM/2/en.md]:
```suggestion
| Russian Federation ![][flag_RU] | 1 | **5** | ![][flag_CN] **China** | [#1](https://osu.ppy.sh/community/matches/59027346) |
```

**Pull request merged**